datafusion/execution/src/runtime_env.rs

```diff
@@ -120,7 +120,7 @@ impl RuntimeEnv {
     ///
     /// # Example: Register local file system object store
```

Comment:
```suggestion
    /// # Example: Register remote URL object store like [Github](https://github.com)
```

Reply:
I'll defer to maintainers, but as a user I'd find that rather confusing given this is providing an example URL string as you would use in code, not a named service with associated link.

I wasn't sure if this was supposed to have a balanced "`" or not, so sent this as removing it (since the link already highlights the example).